features/cucumber.feature

```diff
@@ -256,3 +256,35 @@ Feature: The cucumber command
     # Passing a line number activates serial test execution
     When I run `geordi cucumber --verbose single.feature:1`
     Then the output should contain "> bin/cucumber"
+
+
+  Scenario: Invalid config keys are not reported twice in the same geordi command
+    Given a file named "tmp/global_settings.yml" with "use_vnc: false"
+    Given a file named "features/cucumber.feature" with:
+    """
+    Feature: Running multiple features
+      Scenario: A scenario
+      Scenario: Another scenario
+    """
+
+    When I run `geordi cucumber features/cucumber.feature`
+    Then the output should contain '> Unknown settings in ./tmp/global_settings.yml: use_vnc' 1 time
+      And the output should contain '> Supported settings in ./tmp/global_settings.yml are: ' 1 time
+
+
+  Scenario: Invalid config keys are reported twice in two consecutive executions of geordi
+    Given a file named "tmp/global_settings.yml" with "use_vnc: false"
+    Given a file named "features/cucumber.feature" with:
+    """
+    Feature: Running multiple features
+      Scenario: A scenario
+      Scenario: Another scenario
+    """
+
+    When I run `geordi cucumber features/cucumber.feature`
+    Then the output should contain '> Unknown settings in ./tmp/global_settings.yml: use_vnc' 1 time
+      And the output should contain '> Supported settings in ./tmp/global_settings.yml are: ' 1 time
+
+    When I run `geordi cucumber features/single.feature`
+    Then the output should contain '> Unknown settings in ./tmp/global_settings.yml: use_vnc' 2 times
```

Comment:
In order to avoid misinterpretations, please use the `I ignore previous output` step before L288, and then expect the output again "once".